src/connect.rs

```diff
@@ -72,21 +72,25 @@ pub fn connect_to(peer_info: CrustInfo, send_after_connect: Option<WireMsg>) ->
 
 fn handle_new_connection_res(
     peer_addr: SocketAddr,
-    new_peer_conn_res: Result<quinn::NewConnection, quinn::ConnectionError>,
+    new_peer_conn_res: Result<
+        (
+            quinn::ConnectionDriver,
+            quinn::Connection,
+            quinn::IncomingStreams,
+        ),
+        quinn::ConnectionError,
+    >,
 ) {
-    let new_peer_conn = match new_peer_conn_res {
-        Ok(new_peer_conn) => new_peer_conn,
+    let (conn_driver, q_conn) = match new_peer_conn_res {
+        Ok((conn_driver, conn, _)) => (conn_driver, conn),
```

Comment:
I was reserving `conn` for our `Connection` struct, `new_peer_conn` for `quinn::NewConnection` and `q_conn` for `quinn::Connection`. So let's stick to that pls and retain the name of `new_conn` here.